packages/fxa-auth-server/lib/oauth/routes/authorization.js

```diff
@@ -59,15 +59,11 @@ module.exports = {
         then: Joi.optional(),
         otherwise: Joi.required(),
       }),
-      ttl: Joi.number()
-        .positive()
-        .max(MAX_TTL_S)
-        .default(MAX_TTL_S)
-        .when('response_type', {
-          is: RESPONSE_TYPE_TOKEN,
-          then: Joi.optional(),
-          otherwise: Joi.forbidden(),
-        }),
+      ttl: Joi.number().positive().default(MAX_TTL_S).when('response_type', {
```

Comment:
@6a68 does this also need to be fixed for the `token` endpoint? 

see https://github.com/mozilla/fxa/blob/6a7106a3f218d029899cceaec9e7558e0f7442eb/packages/fxa-auth-server/lib/oauth/routes/token.js#L100

Reply:
Yes, thank you! Discovered that when updating the tests as rfk suggested :-P